packages/main/src/RPA/Browser/Selenium.py

```diff
@@ -603,18 +603,29 @@ def location(self) -> str:
 
     def _find_by_alias(self, parent, criteria, tag, constraints):
         """Custom 'alias' locator that uses locators database."""
-        del constraints
         locator = LocatorsDatabase.load_by_name(criteria, self.locators_path)
 
         if not isinstance(locator, BrowserLocator):
             raise ValueError(f"Not a browser locator: {criteria}")
 
-        selector = "{strategy}:{value}".format(
-            strategy=locator.strategy, value=locator.value
+        strategy = str(locator.strategy).lower()
+        finder = {
+            "class": By.CLASS_NAME,
+            "css": By.CSS_SELECTOR,
+            "id": By.ID,
+            "link": By.LINK_TEXT,
+            "name": By.NAME,
+            "tag": By.TAG_NAME,
+            "xpath": By.XPATH,
+        }[strategy]
```

Comment:
I believe this would raise `KeyError` if key isn't found in the dict, right?
```suggestion
        }.get(strategy)
```